lib/workers/branch/lock-files.js

```diff
@@ -161,6 +189,28 @@ async function writeExistingFiles(config) {
   }
 }
 
+function listLocalLibs(dependencies) {
+  logger.trace(`listLocalLibs (${dependencies})`);
+  const toCopy = [];
+  for (const libName in dependencies) {
+    if ({}.hasOwnProperty.call(dependencies, libName)) {
+      const libVersion = dependencies[libName];
+      if (libVersion.startsWith('file:')) {
+        if (libVersion.endsWith('.tgz')) {
+          logger.error(
```

Comment:
logger.error should be reserved only for serious application errors, i.e. something that requires a `fix:`.
logger.warn should be used when there is "maybe" an application error - not as urgent as error.

In this case, it's a user error, so should be only a logger.info. Instead the way to indicate it to users is via the PR itself.

Reply:
Yes, done